Terraform 0.7.6 crashes importing Azure network resource

Hi there,

Thank you for opening an issue. Please note that we try to keep the Terraform issue tracker reserved for bug reports and feature requests. For general usage questions, please see: https://www.terraform.io/community.html.
### Terraform Version

/var/lib/jenkins/catalina_iaas>terraform -v
Terraform v0.7.6
### Command Run

terraform import azurerm_virtual_network.dev-eu-vnet14 /subscriptions/12dad26c-9b85-4628-9042-xxxxxxxxxxxx/resourceGroups/dev-eu-vnet14-rg/providers/Microsoft.Network/virtualNetworks/dev-eu-vnet14
### Affected Resource(s)

/var/lib/jenkins/catalina_iaas>terraform import azurerm_virtual_network.dev-eu-vnet14 /subscriptions/12dad26c-9b85-4628-9042-xxxxxxxxxxxx/resourceGroups/dev-eu-vnet14-rg/providers/Microsoft.Network/virtualNetworks/dev-eu-vnet14
azurerm_virtual_network.dev-eu-vnet14: Importing from ID "/subscriptions/12dad26c-9b85-4628-9042-xxxxxxxxxxxx/resourceGroups/dev-eu-vnet14-rg/providers/Microsoft.Network/virtualNetworks/dev-eu-vnet14"...
azurerm_virtual_network.dev-eu-vnet14: Import complete!
  Imported azurerm_virtual_network (ID: /subscriptions/12dad26c-9b85-4628-9042-xxxxxxxxxxxx/resourceGroups/dev-eu-vnet14-rg/providers/Microsoft.Network/virtualNetworks/dev-eu-vnet14)
azurerm_virtual_network.dev-eu-vnet14: Refreshing state... (ID: /subscriptions/12dad26c-9b85-4628-9042-xxxxxxxxxxxx/resourceGroups/dev-eu-vnet14-rg/providers/Microsoft.Network/virtualNetworks/dev-eu-vnet14)
Error importing: 1 error(s) occurred:
- azurerm_virtual_network.dev-eu-vnet14: unexpected EOF
  panic: runtime error: invalid memory address or nil pointer dereference
  2016/10/17 17:36:44 [DEBUG] plugin: terraform: [signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x8664838]
  2016/10/17 17:36:44 [DEBUG] plugin: terraform:
  2016/10/17 17:36:44 [DEBUG] plugin: terraform: goroutine 88 [running]:
  2016/10/17 17:36:44 [DEBUG] plugin: terraform: panic(0xa1da340, 0x1bd24008)
  2016/10/17 17:36:44 [DEBUG] plugin: terraform:  /opt/go/src/runtime/panic.go:500 +0x331
  2016/10/17 17:36:44 [DEBUG] plugin: terraform: github.com/hashicorp/terraform/builtin/providers/azurerm.resourceArmVirtualNetworkRead(0x1be36d80, 0xa3b72a0, 0x1bf4c000, 0x0, 0x0)
  2016/10/17 17:36:44 [DEBUG] plugin: terraform:  /opt/gopath/src/github.com/hashicorp/terraform/builtin/providers/azurerm/resource_arm_virtual_network.go:169 +0x888
  2016/10/17 17:36:44 [DEBUG] plugin: terraform: github.com/hashicorp/terraform/helper/schema.(_Resource).Refresh(0x1bf06a50, 0x1c0f3770, 0xa3b72a0, 0x1bf4c000, 0x1bf1a268, 0x0, 0x0)
  2016/10/17 17:36:44 [DEBUG] plugin: terraform:  /opt/gopath/src/github.com/hashicorp/terraform/helper/schema/resource.go:259 +0x2f8
  2016/10/17 17:36:44 [DEBUG] plugin: terraform: github.com/hashicorp/terraform/helper/schema.(_Provider).Refresh(0x1bf143c0, 0x1c072140, 0x1c0f3770, 0x1, 0x0, 0x0)
  2016/10/17 17:36:44 [DEBUG] plugin: terraform:  /opt/gopath/src/github.com/hashicorp/terraform/helper/schema/provider.go:203 +0x140
  2016/10/17 17:36:44 [DEBUG] plugin: terraform: github.com/hashicorp/terraform/plugin.(_ResourceProviderServer).Refresh(0x1bed6b20, 0x1c0bf758, 0x1c0bf900, 0x0, 0x0)
  2016/10/17 17:36:44 [DEBUG] plugin: terraform:  /opt/gopath/src/github.com/hashicorp/terraform/plugin/resource_provider.go:482 +0x57
  2016/10/17 17:36:44 [DEBUG] plugin: terraform: reflect.Value.call(0x1bed3700, 0x1befc478, 0x13, 0xa5c765b, 0x4, 0x1c199f68, 0x3, 0x3, 0x0, 0x0, ...)
  2016/10/17 17:36:44 [DEBUG] plugin: terraform:  /opt/go/src/reflect/value.go:434 +0xdd7
  2016/10/17 17:36:44 [DEBUG] plugin: terraform: reflect.Value.Call(0x1bed3700, 0x1befc478, 0x13, 0x1c199f68, 0x3, 0x3, 0x0, 0x0, 0x0)
  2016/10/17 17:36:44 [DEBUG] plugin: terraform:  /opt/go/src/reflect/value.go:302 +0x8c
  2016/10/17 17:36:44 [DEBUG] plugin: terraform: net/rpc.(_service).call(0x1bf14860, 0x1bed34c0, 0x1bed6e50, 0x1befe410, 0x1bf152c0, 0x9ea7d00, 0x1c0bf758, 0x16, 0x9ea7d40, 0x1c0bf900, ...)
  2016/10/17 17:36:44 [DEBUG] plugin: terraform:  /opt/go/src/net/rpc/server.go:383 +0x12d
  2016/10/17 17:36:44 [DEBUG] plugin: terraform: created by net/rpc.(*Server).ServeCodec
  2016/10/17 17:36:44 [DEBUG] plugin: terraform:  /opt/go/src/net/rpc/server.go:477 +0x33f
  2016/10/17 17:36:44 [ERROR] root: eval: *terraform.EvalRefresh, err: azurerm_virtual_network.dev-eu-vnet14: unexpected EOF
  2016/10/17 17:36:44 [ERROR] root: eval: *terraform.EvalSequence, err: azurerm_virtual_network.dev-eu-vnet14: unexpected EOF
  2016/10/17 17:36:44 [TRACE] [walkImport] Exiting eval tree: import azurerm_virtual_network.dev-eu-vnet14 result: /subscriptions/12dad26c-9b85-4628-9042-e535f3fdcddf/resourceGroups/dev-eu-vnet14-rg/providers/Microsoft.Network/virtualNetworks/dev-eu-vnet14
  2016/10/17 17:36:44 [DEBUG] vertex provider.azurerm (close), got dep: azurerm_virtual_network.dev-eu-vnet14 (import id: /subscriptions/12dad26c-9b85-4628-9042-e535f3fdcddf/resourceGroups/dev-eu-vnet14-rg/providers/Microsoft.Network/virtualNetworks/dev-eu-vnet14)
  2016/10/17 17:36:44 [DEBUG] plugin: waiting for all plugin processes to complete...
  2016/10/17 17:36:44 [DEBUG] plugin: /var/lib/jenkins/terraform_0_7_6/terraform: plugin process exited

!!!!!!!!!!!!!!!!!!!!!!!!!!! TERRAFORM CRASH !!!!!!!!!!!!!!!!!!!!!!!!!!!!

Terraform crashed! This is always indicative of a bug within Terraform.
A crash log has been placed at "crash.log" relative to your current
working directory. It would be immensely helpful if you could please
report the crash with Terraform[1](https://github.com/hashicorp/terraform/issues) so that we can fix this.

When reporting bugs, please include your terraform version. That
information is available on the first line of crash.log. You can also
get it by running 'terraform --version' on the command line.

!!!!!!!!!!!!!!!!!!!!!!!!!!! TERRAFORM CRASH !!!!!!!!!!!!!!!!!!!!!!!!!!!!
### crash.log

/var/lib/jenkins/catalina_iaas>cat crash.log
2016/10/17 17:36:40 [INFO] Terraform version: 0.7.6  754bdda8a78b24efab62f1dc24d97c1c02d89fe9
2016/10/17 17:36:40 [INFO] CLI args: []string{"/var/lib/jenkins/terraform_0_7_6/terraform", "import", "azurerm_virtual_network.dev-eu-vnet14", "/subscriptions/12dad26c-9b85-4628-9042-xxxxxxxxxxxx/resourceGroups/dev-eu-vnet14-rg/providers/Microsoft.Network/virtualNetworks/dev-eu-vnet14"}
2016/10/17 17:36:40 [DEBUG] Detected home directory from env var: /var/lib/jenkins
2016/10/17 17:36:40 [DEBUG] Detected home directory from env var: /var/lib/jenkins
2016/10/17 17:36:40 [DEBUG] Attempting to open CLI config file: /var/lib/jenkins/.terraformrc
2016/10/17 17:36:40 [DEBUG] File doesn't exist, but doesn't need to. Ignoring.
2016/10/17 17:36:40 [TRACE] Preserving existing state lineage "1793bc25-21d7-41dd-8c21-cf6b66ae67a5"
2016/10/17 17:36:40 [TRACE] Preserving existing state lineage "1793bc25-21d7-41dd-8c21-cf6b66ae67a5"
2016/10/17 17:36:40 [TRACE] Graph after step *terraform.ConfigTransformer:

2016/10/17 17:36:40 [TRACE] Graph after step *terraform.ImportStateTransformer:

azurerm_virtual_network.dev-eu-vnet14 (import id: /subscriptions/12dad26c-9b85-4628-9042-xxxxxxxxxxxx/resourceGroups/dev-eu-vnet14-rg/providers/Microsoft.Network/virtualNetworks/dev-eu-vnet14) - *terraform.graphNodeImportState
2016/10/17 17:36:40 [TRACE] Graph after step *terraform.MissingProviderTransformer:

azurerm_virtual_network.dev-eu-vnet14 (import id: /subscriptions/12dad26c-9b85-4628-9042-xxxxxxxxxxxx/resourceGroups/dev-eu-vnet14-rg/providers/Microsoft.Network/virtualNetworks/dev-eu-vnet14) - *terraform.graphNodeImportState
provider.azurerm - *terraform.graphNodeProvider
2016/10/17 17:36:40 [TRACE] Graph after step *terraform.ProviderTransformer:

azurerm_virtual_network.dev-eu-vnet14 (import id: /subscriptions/12dad26c-9b85-4628-9042-xxxxxxxxxxxx/resourceGroups/dev-eu-vnet14-rg/providers/Microsoft.Network/virtualNetworks/dev-eu-vnet14) - *terraform.graphNodeImportState
  provider.azurerm - *terraform.graphNodeProvider
provider.azurerm - *terraform.graphNodeProvider
2016/10/17 17:36:40 [TRACE] Graph after step *terraform.DisableProviderTransformer:

azurerm_virtual_network.dev-eu-vnet14 (import id: /subscriptions/12dad26c-9b85-4628-9042-xxxxxxxxxxxx/resourceGroups/dev-eu-vnet14-rg/providers/Microsoft.Network/virtualNetworks/dev-eu-vnet14) - *terraform.graphNodeImportState
  provider.azurerm - *terraform.graphNodeProvider
provider.azurerm - *terraform.graphNodeProvider
2016/10/17 17:36:40 [TRACE] Graph after step *terraform.PruneProviderTransformer:

azurerm_virtual_network.dev-eu-vnet14 (import id: /subscriptions/12dad26c-9b85-4628-9042-xxxxxxxxxxxx/resourceGroups/dev-eu-vnet14-rg/providers/Microsoft.Network/virtualNetworks/dev-eu-vnet14) - *terraform.graphNodeImportState
  provider.azurerm - *terraform.graphNodeProvider
provider.azurerm - *terraform.graphNodeProvider
2016/10/17 17:36:40 [TRACE] Graph after step *terraform.RootTransformer:

azurerm_virtual_network.dev-eu-vnet14 (import id: /subscriptions/12dad26c-9b85-4628-9042-xxxxxxxxxxxx/resourceGroups/dev-eu-vnet14-rg/providers/Microsoft.Network/virtualNetworks/dev-eu-vnet14) - *terraform.graphNodeImportState
  provider.azurerm - *terraform.graphNodeProvider
provider.azurerm - *terraform.graphNodeProvider
2016/10/17 17:36:40 [TRACE] Graph after step *terraform.CloseProviderTransformer:

azurerm_virtual_network.dev-eu-vnet14 (import id: /subscriptions/12dad26c-9b85-4628-9042-xxxxxxxxxxxx/resourceGroups/dev-eu-vnet14-rg/providers/Microsoft.Network/virtualNetworks/dev-eu-vnet14) - *terraform.graphNodeImportState
  provider.azurerm - *terraform.graphNodeProvider
provider.azurerm - *terraform.graphNodeProvider
provider.azurerm (close) - *terraform.graphNodeCloseProvider
  azurerm_virtual_network.dev-eu-vnet14 (import id: /subscriptions/12dad26c-9b85-4628-9042-xxxxxxxxxxxx/resourceGroups/dev-eu-vnet14-rg/providers/Microsoft.Network/virtualNetworks/dev-eu-vnet14) - *terraform.graphNodeImportState
  provider.azurerm - *terraform.graphNodeProvider
2016/10/17 17:36:40 [TRACE] Graph after step *terraform.TransitiveReductionTransformer:

azurerm_virtual_network.dev-eu-vnet14 (import id: /subscriptions/12dad26c-9b85-4628-9042-xxxxxxxxxxxx/resourceGroups/dev-eu-vnet14-rg/providers/Microsoft.Network/virtualNetworks/dev-eu-vnet14) - _terraform.graphNodeImportState
  provider.azurerm - *terraform.graphNodeProvider
provider.azurerm - *terraform.graphNodeProvider
provider.azurerm (close) - *terraform.graphNodeCloseProvider
  azurerm_virtual_network.dev-eu-vnet14 (import id: /subscriptions/12dad26c-9b85-4628-9042-xxxxxxxxxxxx/resourceGroups/dev-eu-vnet14-rg/providers/Microsoft.Network/virtualNetworks/dev-eu-vnet14) - *terraform.graphNodeImportState
2016/10/17 17:36:40 [DEBUG] Starting graph walk: walkImport
2016/10/17 17:36:40 [DEBUG] Detected home directory from env var: /var/lib/jenkins
2016/10/17 17:36:40 [DEBUG] vertex root.provider.azurerm: walking
2016/10/17 17:36:40 [DEBUG] vertex root.provider.azurerm: evaluating
2016/10/17 17:36:40 [TRACE] [walkImport] Entering eval tree: provider.azurerm
2016/10/17 17:36:40 [DEBUG] root: eval: *terraform.EvalSequence
2016/10/17 17:36:40 [DEBUG] root: eval: *terraform.EvalInitProvider
2016/10/17 17:36:40 [DEBUG] plugin: starting plugin: /var/lib/jenkins/terraform_0_7_6/terraform []string{"/var/lib/jenkins/terraform_0_7_6/terraform", "internal-plugin", "provider", "azurerm"}
2016/10/17 17:36:40 [DEBUG] plugin: waiting for RPC address for: /var/lib/jenkins/terraform_0_7_6/terraform
2016/10/17 17:36:40 [DEBUG] plugin: terraform: 2016/10/17 17:36:40 [INFO] Terraform version: 0.7.6  754bdda8a78b24efab62f1dc24d97c1c02d89fe9
2016/10/17 17:36:40 [DEBUG] plugin: terraform: 2016/10/17 17:36:40 [INFO] CLI args: []string{"/var/lib/jenkins/terraform_0_7_6/terraform", "internal-plugin", "provider", "azurerm"}
2016/10/17 17:36:40 [DEBUG] plugin: terraform: 2016/10/17 17:36:40 [DEBUG] Detected home directory from env var: /var/lib/jenkins
2016/10/17 17:36:40 [DEBUG] plugin: terraform: 2016/10/17 17:36:40 [DEBUG] Detected home directory from env var: /var/lib/jenkins
2016/10/17 17:36:40 [DEBUG] plugin: terraform: 2016/10/17 17:36:40 [DEBUG] Attempting to open CLI config file: /var/lib/jenkins/.terraformrc
2016/10/17 17:36:40 [DEBUG] plugin: terraform: 2016/10/17 17:36:40 [DEBUG] File doesn't exist, but doesn't need to. Ignoring.
2016/10/17 17:36:40 [DEBUG] plugin: terraform: azurerm-provider (internal) 2016/10/17 17:36:40 [INFO] Starting provider plugin azurerm
2016/10/17 17:36:40 [DEBUG] plugin: terraform: azurerm-provider (internal) 2016/10/17 17:36:40 [DEBUG] plugin: plugin address: unix /tmp/plugin670249781
2016/10/17 17:36:40 [DEBUG] plugin: terraform: azurerm-provider (internal) 2016/10/17 17:36:40 [DEBUG] Detected home directory from env var: /var/lib/jenkins
2016/10/17 17:36:40 [DEBUG] root: eval: *terraform.EvalOpFilter
2016/10/17 17:36:40 [DEBUG] root: eval: *terraform.EvalSequence
2016/10/17 17:36:40 [DEBUG] root: eval: *terraform.EvalGetProvider
2016/10/17 17:36:40 [DEBUG] root: eval: *terraform.EvalInterpolate
2016/10/17 17:36:40 [DEBUG] root: eval: *terraform.EvalBuildProviderConfig
2016/10/17 17:36:40 [DEBUG] root: eval: *terraform.EvalInputProvider
2016/10/17 17:36:40 [DEBUG] root: eval: terraform.EvalNoop
2016/10/17 17:36:40 [DEBUG] root: eval: *terraform.EvalOpFilter
2016/10/17 17:36:40 [DEBUG] root: eval: *terraform.EvalSequence
2016/10/17 17:36:40 [DEBUG] root: eval: *terraform.EvalGetProvider
2016/10/17 17:36:40 [DEBUG] root: eval: *terraform.EvalInterpolate
2016/10/17 17:36:40 [DEBUG] root: eval: *terraform.EvalBuildProviderConfig
2016/10/17 17:36:40 [DEBUG] root: eval: *terraform.EvalSetProviderConfig
2016/10/17 17:36:40 [DEBUG] root: eval: *terraform.EvalOpFilter
2016/10/17 17:36:40 [DEBUG] root: eval: *terraform.EvalSequence
2016/10/17 17:36:40 [DEBUG] root: eval: *terraform.EvalConfigProvider
2016/10/17 17:36:43 [TRACE] [walkImport] Exiting eval tree: provider.azurerm
2016/10/17 17:36:43 [DEBUG] vertex azurerm_virtual_network.dev-eu-vnet14 (import id: /subscriptions/12dad26c-9b85-4628-9042-xxxxxxxxxxxx/resourceGroups/dev-eu-vnet14-rg/providers/Microsoft.Network/virtualNetworks/dev-eu-vnet14), got dep: provider.azurerm
2016/10/17 17:36:43 [DEBUG] vertex root.azurerm_virtual_network.dev-eu-vnet14 (import id: /subscriptions/12dad26c-9b85-4628-9042-xxxxxxxxxxxx/resourceGroups/dev-eu-vnet14-rg/providers/Microsoft.Network/virtualNetworks/dev-eu-vnet14): walking
2016/10/17 17:36:43 [DEBUG] vertex root.azurerm_virtual_network.dev-eu-vnet14 (import id: /subscriptions/12dad26c-9b85-4628-9042-xxxxxxxxxxxx/resourceGroups/dev-eu-vnet14-rg/providers/Microsoft.Network/virtualNetworks/dev-eu-vnet14): evaluating
2016/10/17 17:36:43 [TRACE] [walkImport] Entering eval tree: azurerm_virtual_network.dev-eu-vnet14 (import id: /subscriptions/12dad26c-9b85-4628-9042-xxxxxxxxxxxx/resourceGroups/dev-eu-vnet14-rg/providers/Microsoft.Network/virtualNetworks/dev-eu-vnet14)
2016/10/17 17:36:43 [DEBUG] root: eval: *terraform.EvalSequence
2016/10/17 17:36:43 [DEBUG] root: eval: *terraform.EvalGetProvider
2016/10/17 17:36:43 [DEBUG] root: eval: *terraform.EvalImportState
2016/10/17 17:36:43 [TRACE] [walkImport] Exiting eval tree: azurerm_virtual_network.dev-eu-vnet14 (import id: /subscriptions/12dad26c-9b85-4628-9042-xxxxxxxxxxxx/resourceGroups/dev-eu-vnet14-rg/providers/Microsoft.Network/virtualNetworks/dev-eu-vnet14)
2016/10/17 17:36:43 [DEBUG] vertex root.azurerm_virtual_network.dev-eu-vnet14 (import id: /subscriptions/12dad26c-9b85-4628-9042-xxxxxxxxxxxx/resourceGroups/dev-eu-vnet14-rg/providers/Microsoft.Network/virtualNetworks/dev-eu-vnet14): expanding/walking dynamic subgraph
2016/10/17 17:36:43 [DEBUG] vertex root.import azurerm_virtual_network.dev-eu-vnet14 result: /subscriptions/12dad26c-9b85-4628-9042-xxxxxxxxxxxx/resourceGroups/dev-eu-vnet14-rg/providers/Microsoft.Network/virtualNetworks/dev-eu-vnet14: walking
2016/10/17 17:36:43 [DEBUG] vertex root.import azurerm_virtual_network.dev-eu-vnet14 result: /subscriptions/12dad26c-9b85-4628-9042-xxxxxxxxxxxx/resourceGroups/dev-eu-vnet14-rg/providers/Microsoft.Network/virtualNetworks/dev-eu-vnet14: evaluating
2016/10/17 17:36:43 [TRACE] [walkImport] Entering eval tree: import azurerm_virtual_network.dev-eu-vnet14 result: /subscriptions/12dad26c-9b85-4628-9042-xxxxxxxxxxxx/resourceGroups/dev-eu-vnet14-rg/providers/Microsoft.Network/virtualNetworks/dev-eu-vnet14
2016/10/17 17:36:43 [DEBUG] root: eval: *terraform.EvalSequence
2016/10/17 17:36:43 [DEBUG] root: eval: *terraform.EvalGetProvider
2016/10/17 17:36:43 [DEBUG] root: eval: *terraform.EvalRefresh
2016/10/17 17:36:43 [DEBUG] plugin: terraform: azurerm-provider (internal) 2016/10/17 17:36:43 [DEBUG] AzureRM Request:
2016/10/17 17:36:43 [DEBUG] plugin: terraform: GET /subscriptions/12dad26c-9b85-4628-9042-xxxxxxxxxxxx/resourceGroups/dev-eu-vnet14-rg/providers/Microsoft.Network/virtualNetworks/dev-eu-vnet14?api-version=2016-09-01 HTTP/1.1
2016/10/17 17:36:43 [DEBUG] plugin: terraform: Host: management.azure.com
2016/10/17 17:36:43 [DEBUG] plugin: terraform: User-Agent: HashiCorp-Terraform-v0.7.6
2016/10/17 17:36:43 [DEBUG] plugin: terraform: Authorization: Bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiIsIng1dCI6Ikk2b0J3NFZ6QkhPcWxlR3JWMkFKZEE1RW1YYyIsImtpZCI6Ikk2b0J3NFZ6QkhPcWxlR3JWMkFKZEE1RW1YYyJ9.eyJhdWQiOiJodHRwczovL21hbmFnZW1lbnQuYXp1cmUuY29tLyIsImlzcyI6Imh0dHBzOi8vc3RzLndpbmRvd3MubmV0LzJlZGRjMzljLTI5OTYtNGMyYS1hYjk3LWY3NjdjMzllYTE1NS8iLCJpYXQiOjE0NzY3MjU1MDEsIm5iZiI6MTQ3NjcyNTUwMSwiZXhwIjoxNDc2NzI5NDAxLCJhcHBpZCI6IjZkNWY0NGExLTJiMmMtNDZkMy1iNGRmLTYwNDliMDQ0OTdhZiIsImFwcGlkYWNyIjoiMSIsImlkcCI6Imh0dHBzOi8vc3RzLndpbmRvd3MubmV0LzJlZGRjMzljLTI5OTYtNGMyYS1hYjk3LWY3NjdjMzllYTE1NS8iLCJvaWQiOiIxYzI3ZTc2OC0zOWVhLTQ1NGMtYjI4Yy1kMjc4MDc1ZGM4YjgiLCJzdWIiOiIxYzI3ZTc2OC0zOWVhLTQ1NGMtYjI4Yy1kMjc4MDc1ZGM4YjgiLCJ0aWQiOiIyZWRkYzM5Yy0yOTk2LTRjMmEtYWI5Ny1mNzY3YzM5ZWExNTUiLCJ2ZXIiOiIxLjAifQ.RMlTxd9RgUoMHrA7XSEOUO-hv0paI_u3AkVqYrtVuUGjkogRNrwsKmla-zz1mIBvowxcpIYjD9rAVVcE1k27OrQaPD7tFFZetBHWq1OEf9saWUE_ftMgLAse6aVOaWwMNfOG6IX1Cr4oKvCvW2Wql2HSR1RlnUaiXOVROfKqVwjhu6BPC8iSWV9ynjVsSGGp_R8kMcTHR_ALV6noid94cW7yi_x8Bnprng_ImZpKpZQe9Jee_VtEOpKdfP3g5TypZhAk3-zfMct3ikWWH7CPUbDn5Kl1eicAO6yci58l6beeQjEMG9P11rKwQllM8WX8c_H6TS5nkfJXBuC8zFAk0g
2016/10/17 17:36:43 [DEBUG] plugin: terraform: Accept-Encoding: gzip
2016/10/17 17:36:43 [DEBUG] plugin: terraform:
2016/10/17 17:36:43 [DEBUG] plugin: terraform:
2016/10/17 17:36:44 [DEBUG] plugin: terraform: azurerm-provider (internal) 2016/10/17 17:36:44 [DEBUG] AzureRM Response for https://management.azure.com/subscriptions/12dad26c-9b85-4628-9042-xxxxxxxxxxxx/resourceGroups/dev-eu-vnet14-rg/providers/Microsoft.Network/virtualNetworks/dev-eu-vnet14?api-version=2016-09-01:
2016/10/17 17:36:44 [DEBUG] plugin: terraform: HTTP/1.1 200 OK
2016/10/17 17:36:44 [DEBUG] plugin: terraform: Transfer-Encoding: chunked
2016/10/17 17:36:44 [DEBUG] plugin: terraform: Cache-Control: no-cache
2016/10/17 17:36:44 [DEBUG] plugin: terraform: Content-Type: application/json; charset=utf-8
2016/10/17 17:36:44 [DEBUG] plugin: terraform: Date: Mon, 17 Oct 2016 17:36:43 GMT
2016/10/17 17:36:44 [DEBUG] plugin: terraform: Etag: W/"4d28f7ac-4201-4cb0-9086-412f50190c70"
2016/10/17 17:36:44 [DEBUG] plugin: terraform: Expires: -1
2016/10/17 17:36:44 [DEBUG] plugin: terraform: Pragma: no-cache
2016/10/17 17:36:44 [DEBUG] plugin: terraform: Server: Microsoft-HTTPAPI/2.0
2016/10/17 17:36:44 [DEBUG] plugin: terraform: Server: Microsoft-HTTPAPI/2.0
2016/10/17 17:36:44 [DEBUG] plugin: terraform: Strict-Transport-Security: max-age=31536000; includeSubDomains
2016/10/17 17:36:44 [DEBUG] plugin: terraform: Vary: Accept-Encoding
2016/10/17 17:36:44 [DEBUG] plugin: terraform: X-Ms-Correlation-Request-Id: 8780c040-2b0c-41fa-be95-09853aab01c9
2016/10/17 17:36:44 [DEBUG] plugin: terraform: X-Ms-Ratelimit-Remaining-Subscription-Reads: 14995
2016/10/17 17:36:44 [DEBUG] plugin: terraform: X-Ms-Request-Id: 9e96ff68-8324-4abb-a59d-f415039f3d9a
2016/10/17 17:36:44 [DEBUG] plugin: terraform: X-Ms-Routing-Request-Id: EASTUS:20161017T173644Z:8780c040-2b0c-41fa-be95-09853aab01c9
2016/10/17 17:36:44 [DEBUG] plugin: terraform:
2016/10/17 17:36:44 [DEBUG] plugin: terraform: 71d
2016/10/17 17:36:44 [DEBUG] plugin: terraform: {
2016/10/17 17:36:44 [DEBUG] plugin: terraform:   "name": "dev-eu-vnet14",
2016/10/17 17:36:44 [DEBUG] plugin: terraform:   "id": "/subscriptions/12dad26c-9b85-4628-9042-xxxxxxxxxxxx/resourceGroups/dev-eu-vnet14-rg/providers/Microsoft.Network/virtualNetworks/dev-eu-vnet14",
2016/10/17 17:36:44 [DEBUG] plugin: terraform:   "etag": "W/\"4d28f7ac-4201-4cb0-9086-412f50190c70\"",
2016/10/17 17:36:44 [DEBUG] plugin: terraform:   "type": "Microsoft.Network/virtualNetworks",
2016/10/17 17:36:44 [DEBUG] plugin: terraform:   "location": "eastus",
2016/10/17 17:36:44 [DEBUG] plugin: terraform:   "properties": {
2016/10/17 17:36:44 [DEBUG] plugin: terraform:     "provisioningState": "Succeeded",
2016/10/17 17:36:44 [DEBUG] plugin: terraform:     "resourceGuid": "2fbc5abf-a759-447b-97db-484203d15a0b",
2016/10/17 17:36:44 [DEBUG] plugin: terraform:     "addressSpace": {
2016/10/17 17:36:44 [DEBUG] plugin: terraform:       "addressPrefixes": [
2016/10/17 17:36:44 [DEBUG] plugin: terraform:         "10.166.14.0/24"
2016/10/17 17:36:44 [DEBUG] plugin: terraform:       ]
2016/10/17 17:36:44 [DEBUG] plugin: terraform:     },
2016/10/17 17:36:44 [DEBUG] plugin: terraform:     "subnets": [
2016/10/17 17:36:44 [DEBUG] plugin: terraform:       {
2016/10/17 17:36:44 [DEBUG] plugin: terraform:         "name": "GatewaySubnet",
2016/10/17 17:36:44 [DEBUG] plugin: terraform:         "id": "/subscriptions/12dad26c-9b85-4628-9042-xxxxxxxxxxxx/resourceGroups/dev-eu-vnet14-rg/providers/Microsoft.Network/virtualNetworks/dev-eu-vnet14/subnets/GatewaySubnet",
2016/10/17 17:36:44 [DEBUG] plugin: terraform:         "etag": "W/\"4d28f7ac-4201-4cb0-9086-412f50190c70\"",
2016/10/17 17:36:44 [DEBUG] plugin: terraform:         "properties": {
2016/10/17 17:36:44 [DEBUG] plugin: terraform:           "provisioningState": "Succeeded",
2016/10/17 17:36:44 [DEBUG] plugin: terraform:           "addressPrefix": "10.166.14.0/28",
2016/10/17 17:36:44 [DEBUG] plugin: terraform:           "ipConfigurations": [
2016/10/17 17:36:44 [DEBUG] plugin: terraform:             {
2016/10/17 17:36:44 [DEBUG] plugin: terraform:               "id": "/subscriptions/12dad26c-9b85-4628-9042-xxxxxxxxxxxx/resourceGroups/dev-eu-vnet14-rg/providers/Microsoft.Network/virtualNetworkGateways/dev-eu-vnet14-vgw/ipConfigurations/dev-eu-vnet14-vgwpipconfig"
2016/10/17 17:36:44 [DEBUG] plugin: terraform:             }
2016/10/17 17:36:44 [DEBUG] plugin: terraform:           ],
2016/10/17 17:36:44 [DEBUG] plugin: terraform:           "serviceTunnels": []
2016/10/17 17:36:44 [DEBUG] plugin: terraform:         }
2016/10/17 17:36:44 [DEBUG] plugin: terraform:       },
2016/10/17 17:36:44 [DEBUG] plugin: terraform:       {
2016/10/17 17:36:44 [DEBUG] plugin: terraform:         "name": "dev-eu-vnet14-subnet1",
2016/10/17 17:36:44 [DEBUG] plugin: terraform:         "id": "/subscriptions/12dad26c-9b85-4628-9042-xxxxxxxxxxxx/resourceGroups/dev-eu-vnet14-rg/providers/Microsoft.Network/virtualNetworks/dev-eu-vnet14/subnets/dev-eu-vnet14-subnet1",
2016/10/17 17:36:44 [DEBUG] plugin: terraform:         "etag": "W/\"4d28f7ac-4201-4cb0-9086-412f50190c70\"",
2016/10/17 17:36:44 [DEBUG] plugin: terraform:         "properties": {
2016/10/17 17:36:44 [DEBUG] plugin: terraform:           "provisioningState": "Succeeded",
2016/10/17 17:36:44 [DEBUG] plugin: terraform:           "addressPrefix": "10.166.14.64/26",
2016/10/17 17:36:44 [DEBUG] plugin: terraform:           "serviceTunnels": []
2016/10/17 17:36:44 [DEBUG] plugin: terraform:         }
2016/10/17 17:36:44 [DEBUG] plugin: terraform:       }
2016/10/17 17:36:44 [DEBUG] plugin: terraform:     ],
2016/10/17 17:36:44 [DEBUG] plugin: terraform:     "virtualNetworkPeerings": []
2016/10/17 17:36:44 [DEBUG] plugin: terraform:   }
2016/10/17 17:36:44 [DEBUG] plugin: terraform: }
2016/10/17 17:36:44 [DEBUG] plugin: terraform: 0
2016/10/17 17:36:44 [DEBUG] plugin: terraform:
2016/10/17 17:36:44 [DEBUG] plugin: terraform:
2016/10/17 17:36:44 [DEBUG] plugin: terraform: panic: runtime error: invalid memory address or nil pointer dereference
2016/10/17 17:36:44 [DEBUG] plugin: terraform: [signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x8664838]
2016/10/17 17:36:44 [DEBUG] plugin: terraform:
2016/10/17 17:36:44 [DEBUG] plugin: terraform: goroutine 88 [running]:
2016/10/17 17:36:44 [DEBUG] plugin: terraform: panic(0xa1da340, 0x1bd24008)
2016/10/17 17:36:44 [DEBUG] plugin: terraform:  /opt/go/src/runtime/panic.go:500 +0x331
2016/10/17 17:36:44 [DEBUG] plugin: terraform: github.com/hashicorp/terraform/builtin/providers/azurerm.resourceArmVirtualNetworkRead(0x1be36d80, 0xa3b72a0, 0x1bf4c000, 0x0, 0x0)
2016/10/17 17:36:44 [DEBUG] plugin: terraform:  /opt/gopath/src/github.com/hashicorp/terraform/builtin/providers/azurerm/resource_arm_virtual_network.go:169 +0x888
2016/10/17 17:36:44 [DEBUG] plugin: terraform: github.com/hashicorp/terraform/helper/schema.(_Resource).Refresh(0x1bf06a50, 0x1c0f3770, 0xa3b72a0, 0x1bf4c000, 0x1bf1a268, 0x0, 0x0)
2016/10/17 17:36:44 [DEBUG] plugin: terraform:  /opt/gopath/src/github.com/hashicorp/terraform/helper/schema/resource.go:259 +0x2f8
2016/10/17 17:36:44 [DEBUG] plugin: terraform: github.com/hashicorp/terraform/helper/schema.(_Provider).Refresh(0x1bf143c0, 0x1c072140, 0x1c0f3770, 0x1, 0x0, 0x0)
2016/10/17 17:36:44 [DEBUG] plugin: terraform:  /opt/gopath/src/github.com/hashicorp/terraform/helper/schema/provider.go:203 +0x140
2016/10/17 17:36:44 [DEBUG] plugin: terraform: github.com/hashicorp/terraform/plugin.(_ResourceProviderServer).Refresh(0x1bed6b20, 0x1c0bf758, 0x1c0bf900, 0x0, 0x0)
2016/10/17 17:36:44 [DEBUG] plugin: terraform:  /opt/gopath/src/github.com/hashicorp/terraform/plugin/resource_provider.go:482 +0x57
2016/10/17 17:36:44 [DEBUG] plugin: terraform: reflect.Value.call(0x1bed3700, 0x1befc478, 0x13, 0xa5c765b, 0x4, 0x1c199f68, 0x3, 0x3, 0x0, 0x0, ...)
2016/10/17 17:36:44 [DEBUG] plugin: terraform:  /opt/go/src/reflect/value.go:434 +0xdd7
2016/10/17 17:36:44 [DEBUG] plugin: terraform: reflect.Value.Call(0x1bed3700, 0x1befc478, 0x13, 0x1c199f68, 0x3, 0x3, 0x0, 0x0, 0x0)
2016/10/17 17:36:44 [DEBUG] plugin: terraform:  /opt/go/src/reflect/value.go:302 +0x8c
2016/10/17 17:36:44 [DEBUG] plugin: terraform: net/rpc.(_service).call(0x1bf14860, 0x1bed34c0, 0x1bed6e50, 0x1befe410, 0x1bf152c0, 0x9ea7d00, 0x1c0bf758, 0x16, 0x9ea7d40, 0x1c0bf900, ...)
2016/10/17 17:36:44 [DEBUG] plugin: terraform:  /opt/go/src/net/rpc/server.go:383 +0x12d
2016/10/17 17:36:44 [DEBUG] plugin: terraform: created by net/rpc.(_Server).ServeCodec
2016/10/17 17:36:44 [DEBUG] plugin: terraform:  /opt/go/src/net/rpc/server.go:477 +0x33f
2016/10/17 17:36:44 [ERROR] root: eval: *terraform.EvalRefresh, err: azurerm_virtual_network.dev-eu-vnet14: unexpected EOF
2016/10/17 17:36:44 [ERROR] root: eval: *terraform.EvalSequence, err: azurerm_virtual_network.dev-eu-vnet14: unexpected EOF
2016/10/17 17:36:44 [TRACE] [walkImport] Exiting eval tree: import azurerm_virtual_network.dev-eu-vnet14 result: /subscriptions/12dad26c-9b85-4628-9042-xxxxxxxxxxxx/resourceGroups/dev-eu-vnet14-rg/providers/Microsoft.Network/virtualNetworks/dev-eu-vnet14
2016/10/17 17:36:44 [DEBUG] vertex provider.azurerm (close), got dep: azurerm_virtual_network.dev-eu-vnet14 (import id: /subscriptions/12dad26c-9b85-4628-9042-xxxxxxxxxxxx/resourceGroups/dev-eu-vnet14-rg/providers/Microsoft.Network/virtualNetworks/dev-eu-vnet14)
2016/10/17 17:36:44 [DEBUG] plugin: waiting for all plugin processes to complete...
2016/10/17 17:36:44 [DEBUG] plugin: /var/lib/jenkins/terraform_0_7_6/terraform: plugin process exited
/var/lib/jenkins/catalina_iaas>

--- a/builtin/providers/azurerm/resource_arm_virtual_network.go
+++ b/builtin/providers/azurerm/resource_arm_virtual_network.go
@@ -165,11 +165,13 @@ func resourceArmVirtualNetworkRead(d *schema.ResourceData, meta interface{}) err
 	}
 	d.Set("subnet", subnets)
 
-	dnses := []string{}
-	for _, dns := range *vnet.DhcpOptions.DNSServers {
-		dnses = append(dnses, dns)
+	if vnet.DhcpOptions != nil && vnet.DhcpOptions.DNSServers != nil {
+		dnses := []string{}
+		for _, dns := range *vnet.DhcpOptions.DNSServers {
+			dnses = append(dnses, dns)
+		}
+		d.Set("dns_servers", dnses)
 	}
-	d.Set("dns_servers", dnses)
 
 	flattenAndSetTags(d, resp.Tags)
 

